core/src/main/java/dev/morphia/aggregation/Aggregation.java

```diff
@@ -1,32 +1,13 @@
 package dev.morphia.aggregation;
 
+import java.util.List;
+
 import dev.morphia.aggregation.expressions.impls.DocumentExpression;
 import dev.morphia.aggregation.expressions.impls.Expression;
-import dev.morphia.aggregation.stages.AddFields;
-import dev.morphia.aggregation.stages.AutoBucket;
-import dev.morphia.aggregation.stages.Bucket;
-import dev.morphia.aggregation.stages.ChangeStream;
-import dev.morphia.aggregation.stages.CollectionStats;
-import dev.morphia.aggregation.stages.CurrentOp;
-import dev.morphia.aggregation.stages.Densify;
-import dev.morphia.aggregation.stages.Facet;
-import dev.morphia.aggregation.stages.Fill;
+import dev.morphia.aggregation.stages.*;
```

Comment:
* imports break checkstyle so you'll want to clean that up.  The code formatter that runs as part of the maven build *should* clean that up but if not, you'll need to manually expand those.